docs/content/concepts/logging.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Logging | Dagster Docs"
+description: "Dagster includes a rich and extensible logging system"
+---
+
+# Logging
+
+Dagster supports a variety of built-in logging options, as well as the ability to extend and customize them. By default, Dagster automatically tracks and captures all execution events, such as those during an asset materialization. During these events, logs like the following are generated:
+
+```bash
+12:40:05 - DEBUG - RUN_START - Started execution of run for "__ASSET_JOB_0".
+12:40:05 - DEBUG - ENGINE_EVENT - Executing steps using multiprocess executor: parent process (pid: 86387)
+12:40:05 - DEBUG - taxi_zones_file - STEP_WORKER_STARTING - Launching subprocess for "taxi_zones_file".
+12:40:07 - DEBUG - STEP_WORKER_STARTED - Executing step "taxi_zones_file" in subprocess.
+12:40:07 - DEBUG - taxi_zones_file - RESOURCE_INIT_S TARTED - Starting initialization of resources [io_manager].
+12:40:07 - DEBUG - taxi_zones_file - RESOURCE_INIT_SUCCESS - Finished initialization of resources [io_manager].
+12:40:07 - DEBUG - LOGS_CAPTURED - Started capturing logs in process (pid: 86390).
+12:40:07 - DEBUG - taxi_zones_file - STEP_START - Started execution of step "taxi_zones_file".
+12:40:09 - DEBUG - taxi_zones_file - STEP_OUTPUT - Yielded output "result" of type "Any". (Type check passed).
+12:40:09 - DEBUG - __ASSET_JOB_0 - taxi_zones_file - Writing file at: /Users/erincochran/Desktop/dagster-examples/project-dagster-university/tmpfxsoltsc/storage/taxi_zones_file using PickledObjectFilesystemIOManager...
+12:40:09 - DEBUG - taxi_zones_file - ASSET_MATERIALIZATION - Materialized value taxi_zones_file.
+12:40:09 - DEBUG - taxi_zones_file - HANDLED_OUTPUT - Handled output "result" using IO manager "io_manager"
+12:40:09 - DEBUG - taxi_zones_file - STEP_SUCCESS - Finished execution of step "taxi_zones_file" in 1.17s.
+12:40:09 - DEBUG - ENGINE_EVENT - Multiprocess executor: parent process exiting after 4.38s (pid: 86387)
+12:40:09 - DEBUG - RUN_SUCCESS - Finished execution of run for "__ASSET_JOB_0".
+```
+
+These logs can be exported to a local file or viewed in the [Dagster UI](#logs-in-the-dagster-ui).
+
+---
+
+## Benefits
```

Comment:
Including a benefits section here feels a little awkward, because logging is such tablestakes functionality of any system.  Not totally sure this is a fair comparison, but it feels a little bit like if a brochure for a car had a section on steering and that section had a "benefits" subsection with points like "you can make turns".

Very possible I'm making too many assumptions here about the perspective that our users are coming from, so consider this comment more of an attempt to start a conversation on this than a strongly-held opinion.

Reply:
I know; I agree it felt weird. I was aiming to keep this consistent with the approach taken in some other newly-revamped concept pages, but it feels like a stretch here.

I've removed this and replaced it with a section about log types instead.